packages/scripts/src/monitor-oa/common.ts

```diff
@@ -0,0 +1,111 @@
+import { getRetryProvider } from "@uma/common";
+import { ERC20Ethers } from "@uma/contracts-node";
+import { delay } from "@uma/financial-templates-lib";
+import { utils } from "ethers";
+import { getContractInstanceWithProvider } from "../utils/contracts";
+
+import type { Provider } from "@ethersproject/abstract-provider";
+
+export interface BotModes {
+  assertionsEnabled: boolean;
+  disputesEnabled: boolean;
+  settlementsEnabled: boolean;
+}
+
+export interface BlockRange {
+  start: number;
+  end: number;
+}
+
+export interface MonitoringParams {
+  provider: Provider;
+  chainId: number;
+  blockRange: BlockRange;
+  pollingDelay: number;
+  botModes: BotModes;
+}
+
+export const initMonitoringParams = async (env: NodeJS.ProcessEnv): Promise<MonitoringParams> => {
+  if (!env.CHAIN_ID) throw new Error("CHAIN_ID must be defined in env");
+  const chainId = Number(env.CHAIN_ID);
+
+  if (!chainId || (chainId != 1 && chainId != 5)) throw new Error("This script should be run on mainnet or goerli");
```

Comment:
We can support more networks than 1 and 5 right?
https://github.com/UMAprotocol/protocol/commit/6fddb23c8a62542084d6d72fdfab00213ffdcd8d